u8timeseries/timeseries.py

```diff
@@ -20,35 +19,34 @@
 
 class TimeSeries:
     def __init__(self,
-                 series: pd.Series,
-                 confidence_lo: Optional[pd.Series] = None,
-                 confidence_hi: Optional[pd.Series] = None,
+                 series: Union[pd.Series, pd.DataFrame],
                  freq: Optional[str] = None,
                  fill_missing_dates: Optional[bool] = True):
         """
-        A TimeSeries is an object representing a univariate time series, and optional confidence intervals.
+        A TimeSeries is an object representing a time series.
 
         TimeSeries are meant to be immutable.
 
         Parameters
         ----------
         series
-            The actual time series, as a pandas Series with a proper time index.
-        confidence_lo
-            Optionally, a Pandas Series representing lower confidence interval.
-        confidence_hi
-            Optionally, a Pandas Series representing upper confidence interval.
+            The actual time series, as a pandas Series or DataFrame with a proper time index.
         freq
             Optionally, a string representing the frequency of the Pandas Series. When creating a TimeSeries
             instance with a length smaller than 3, this argument must be passed.
         fill_missing_dates
             Optionally, a boolean value indicating whether to fill missing dates with NaN values
             in case the frequency of `series` cannot be inferred.
         """
+        # create dataframes
+        if (isinstance(series, pd.Series)):
+            series = pd.DataFrame(series)
+        series.columns = range(series.shape[1])
 
-        raise_if_not(len(series) > 0, 'Series must not be empty.', logger)
+        raise_if_not(len(series) > 0 and series.shape[1] > 0, 'Series must not be empty.', logger)
```

Comment:
`Dataframe must not be empty`

Reply:
My thinking was that although we do not use a pandas series as underlying data type anymore, we are still talking about time series. One (maybe minor) issue that I can see happening if we replace 'Series' with 'Dataframe' in this error message is that if this error is thrown by the user calling TimeSeries.from_series(pd_series) where pd_series is empty, the error message will be 'Dataframe must not be empty' which might confuse the user.

Reply:
Good point. How about `Time series must not be empty` ?

Reply:
Yeah that sounds like a good solution

Reply:
done